authentication/views.py

```diff
@@ -2,10 +2,51 @@
 Views for authentication. Basically supports login/logout.
 """
 from django.shortcuts import render
+from django.http import HttpResponseRedirect, HttpResponse
+from django.contrib.auth import logout as log_out
+from django.contrib.auth.decorators import login_required
+import plaid
+from authentication.models import UserBank
+
+
+PLAID_CLIENT_ID = '59c3248dbdc6a40ac87ed3e8'
+PLAID_SECRET = 'ee7f10de9feb7d6408d1f5bd980f2a'
```

Comment:
This project is currently publically viewable. You will have to remove this and inject it as environment variables instead. Since the secret is already public through this commit, please delete the access tokens from the PLAID account and create new ones

Reply:
Yea, I left these in because A. these are sandbox only (it has literally no access to anything right now) and we'll need to change them anyways when we get development access (where we can use actual WF data), B. that way others can test.
When we get development access, I'll switch them out for env variables.